jrt/src/com/yahoo/jrt/TransportMetrics.java

```diff
@@ -0,0 +1,79 @@
+// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.jrt;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+/**
+ * Metric values produced by {@link Transport}.
```

Comment:
these metrics seem more crypto related and not so much transport... maybe use crypto/tls in the name instead of transport (similar to C++ version)